apps/salonchroma/manifest.yaml

```diff
@@ -0,0 +1,8 @@
+---
+id: salonchroma
+name: SalonChroma
+summary: Show Logo for Salon Example
```

Comment:
Might be easier to just use the "photos" app and upload a custom graphic that way?

Reply:
I would agree with you here. I didn't know about the photo app but see it now. Also, I am able to use the Api to push it as well. I was just including it to show folks how to simply show an image in its simplest form.